src/types/configData.go

```diff
@@ -111,11 +111,19 @@ type ConfigPublisher struct {
 }
 
 type ConfigPurgeOldDBEntries struct {
-	Enable              bool     `json:"enable,omitempty" bson:"enable,omiempty"`
+	Enable              bool     `json:"enable,omitempty" bson:"enable,omitempty"`
 	CronJobTimeInterval string   `json:"cronjob_time_interval,omitempty" bson:"cronjob_time_interval,omitempty"`
 	DBName              []string `json:"db_name,omitempty" bson:"db_name,omitempty"`
 }
 
+type ConfigRecommendPolicy struct {
+	Enable                  bool   `json:"enable,omitempty" bson:"enable,omitempty"`
+	OperationMode           int    `json:"operation_mode,omitempty" bson:"operation_mode,omitempty"`
+	CronJobTimeInterval     string `json:"cronjob_time_interval,omitempty" bson:"cronjob_time_interval,omitempty"`
+	DBName                  string `json:"db_name,omitempty" bson:"db_name,omitempty"`
```

Comment:
Why is DBName another configuration here? We should be using the same DB where we keep discovered policies.